cfpb/cfgov-refresh
# Mega Menu: Remove variation 2 mega menu

Removes the tested variation 2 mega menu added in https://github.com/cfpb/cfgov-refresh/pull/5428

## Removals

- Remove variation 2 mega menu and `MEGA_MENU_VAR_2` flag.

## Testing

1. Pull branch and run `gulp clean && gulp build`
2. Enable the `MEGA_MENU_VAR_1` flag and reload the homepage and see that the menu is the variation 1 menu.
3. Test closing the menu by click off of the menu in various spots. Menu should not close when clicking in the menu or when the mouse leaves the web page window.
